adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -267,7 +267,7 @@ class ADAMContext private (@transient val sc: SparkContext) extends Serializable
    * @tparam T The type of records to return
    * @return An RDD with records of the specified type
    */
-  private[rdd] def loadParquet[T](
+  def loadParquet[T](
```

Comment:
It seems like this would be useful outside of ADAM.  I haven't fully thought through the use case though: someone wants to add a new schema record `Foo`, they extend `ADAMKyroRegistrator` to register, then extend `ADAMContext` to add their new `loadFoo` method, which presumably would call `loadParquet` for `foo.adam` paths.  If `Foo` has a sequence dictionary or samples, those avro metadata methods would also be useful, and extending from `GenomicRDD` and friends needs to be possible.